db/migrate/20250127115114_create_admins.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAdmins < ActiveRecord::Migration[7.2]
+  def change
+    create_table :admins do |t|
```

Comment:
Instead of creating seperate table, you can add `role` column in `users` table where role can be admin, etc